src/collective/upload/browser/static/upload.js

```diff
@@ -0,0 +1,16 @@
+* http://jqueryui.com
+* Includes: widget.js
+* Copyright 2015 jQuery Foundation and other contributors; Licensed MIT */
+!function(a){r=[i(1)],n=a,void 0!==(o="function"==typeof n?n.apply(t,r):n)&&(e.exports=o)}(function(e){/*!
+ * jQuery UI Widget 1.11.4
```

Comment:
no, it just includes `jquery.ui.widget`.

I try to take this script out, but for some reason `jquery.fileupload` depends on it, and webpack does the favor to insert into generated JS: https://github.com/blueimp/jQuery-File-Upload/blob/master/js/jquery.fileupload.js#L27